main/settings.py

```diff
@@ -88,6 +88,18 @@
     description="Application-level SSL redirect setting.",
 )
 
+
+USE_X_FORWARDED_HOST = get_bool(
+    name="USE_X_FORWARDED_HOST",
+    default=False,
```

Comment:
Minor nitpick - any harm in setting the default to True instead?  Otherwise will need to make sure they're set to True in saltops.

Reply:
This will break environments where that header isn't set so leaving it off (django's default) is a safer default.